src/test/java/org/example/homework13/blackjack/DeckTest.java

```diff
@@ -0,0 +1,44 @@
+package org.example.homework13.blackjack;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class DeckTest {
+
+    @Test
+    void drawCard() {
+        Deck deck = new Deck();
+        Card card = deck.drawCard();
+        assertEquals(Rank.TWO, card.getRank());
+        assertEquals(Suit.CLUBS, card.getSuit());
+    }
+
+    @Test
+    void shuffle() {
+        Deck referenceDeck = new Deck();
+        Deck shuffledDeck = new Deck();
+        shuffledDeck.shuffle();
+        boolean diffFound = false;
+        for (int i = 0; i < 52; i++) {
```

Comment:
when you will compare two listst of cards(see previous comment) you won't need to use loop, just compare 2 listst

Reply:
removed loop